gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyLogging.java

```diff
@@ -81,4 +81,15 @@ public  void generateSummaryEvents(DichotomyDirection direction, DichotomyResult
     private static String toString(Collection<String> c) {
         return c.stream().map(Object::toString).collect(Collectors.joining(", "));
     }
+
+    private String getVoltageCheckResult(DichotomyDirection direction, Optional<VoltageMonitoringResult> voltageMonitoringResult) {
+        if (direction.equals(DichotomyDirection.FR_ES) || direction.equals(DichotomyDirection.ES_FR)) {
+            if (voltageMonitoringResult.isPresent()) {
+                return String.valueOf(voltageMonitoringResult.get().getStatus());
```

Comment:
Can .getStatus() return null ? If yes, you could add something to handle the error

Reply:
No it can not be null